### PR TITLE
Doc/frenzy 2

### PR DESCRIPTION
**Pull request by mr-c**
re-do  of https://github.com/dib-lab/khmer/pull/1241

### CODE REVIEW

File: doc/dev/coding-guidelines-and-review.rst [hunk 1]
```diff
@@ -109,6 +109,7 @@ ready for review::
      http://en.wikipedia.org/wiki/Changelog#Format
    - [ ] Was a spellchecker run on the source code and documentation after
      changes were made?
+   - [ ] Does the script respect streaming IO? (Is it tested for streaming IO?)
```
Comment (mr-c) [inline doc/dev/coding-guidelines-and-review.rst]: Some pull requests don't have scripts, therefore 'the script' doesn't make sense.

File: doc/LICENSE.rst [hunk 2]
```diff
@@ -5,6 +5,7 @@ License
 =======
 
 Copyright (c) 2010-2014, Michigan State University.  All rights reserved.
```
Comment (mr-c) [inline doc/LICENSE.rst]: Turns out we can drop the 'all rights reserved'

File: doc/dev/binary-file-formats.rst [hunk 3]
```diff
@@ -56,6 +56,7 @@ Use Bigcount        1       6   ``0x01`` if bigcounts is used, else ``0x00``
 K-size              4       7   k-mer length, ``ht._ksize``. [``uint32_t``]
 Number of Tables    1      11   Number of Count-min Sketch tables,
                                 ``ht._n_tables``. [``uint8_t``]
+Occupied Bins       8      12   Number of occupied bins
```
Comment (mr-c) [inline doc/dev/binary-file-formats.rst]: Can you update the preferred extensions as well?

File: doc/dev/codebase-guide.rst [hunk 4]
```diff
@@ -9,10 +9,10 @@ The ChangeLog file lists changes to the codebase, most recent first.
 
 The lib/ directory contains all of the C++ code.
```
Comment (mr-c) [inline doc/dev/codebase-guide.rst]: lib should also be stylized as below

File: doc/dev/codebase-guide.rst [hunk 5]
```diff
@@ -9,10 +9,10 @@ The ChangeLog file lists changes to the codebase, most recent first.
...
 
-The khmer/ directory contains the khmer package (khmer/__init__.py etc)
-and the C++-to-Python bridge (khmer/_khmermodule.cc).
+The `khmer/` directory contains the `khmer` package (`khmer/__init__.py`, etc)
```
Comment (mr-c) [inline doc/dev/codebase-guide.rst]: single backticks are for italics; double backticks for code please

File: doc/dev/development.rst [hunk 6]
```diff
@@ -54,13 +54,13 @@ Pipelines
 ---------
 
 All khmer scripts used by a published recommended analysis pipeline must be
-included in scripts/ and meet the standards therein implied.
+included in `scripts/` and meet the standards therein implied.
```
Comment (mr-c) [inline doc/dev/development.rst]: please also use double backticks in this file for code + pathname snippets

File: doc/dev/getting-started.rst [hunk 7]
```diff
@@ -242,9 +269,42 @@ Claiming an issue and starting to develop
...
+   previously in the repository but git has noticed.
+
+   To commit changes you have to 'stage' them--this is done by issuing the
+   following command::
+
+      git add path/to/file
+
+   If you have a large quanity of changes and you don't want to add each file
+   manually you can do `git add --all`--but be careful! This can sometimes add
+   changes you didn't intend to commit, such as extraneous test datia--You can
```
Comment (mr-c) [inline doc/dev/getting-started.rst]: daita? :-)

File: doc/dev/getting-started.rst [hunk 8]
```diff
@@ -242,9 +269,42 @@ Claiming an issue and starting to develop
...
+   changed. A file in the 'untracked' section are files that haven't existed
+   previously in the repository but git has noticed.
+
+   To commit changes you have to 'stage' them--this is done by issuing the
+   following command::
+
+      git add path/to/file
+
+   If you have a large quanity of changes and you don't want to add each file
+   manually you can do `git add --all`--but be careful! This can sometimes add
```
Comment (mr-c) [inline doc/dev/getting-started.rst]: mention `git add --patch` instead of `git add --all`

Comment (mr-c) [inline doc/dev/getting-started.rst]: spaces around your emdash, please

File: doc/dev/getting-started.rst [hunk 9]
```diff
@@ -242,9 +269,42 @@ Claiming an issue and starting to develop
...
+   changes you've made. Please make your commit message informative but
+   concise--these messages become part of the 'official' history of the
+   project. 
+
+   Once your changes have been committed, push them up to the remote branch::
+
+      git push
+
+   If this is your first commit on a new branch git will error out, telling
+   you the remote branch doesn't exist--This is fine, as it will also provide
```
Comment (mr-c) [inline doc/dev/getting-started.rst]: spaces around your emdash, please

File: doc/dev/scripts-and-sandbox.rst [hunk 10]
```diff
@@ -78,6 +78,15 @@ Our current Copyright message is::
    # Contact: khmer-project@idyll.org
    #
 
+Some files are copyright University of California Regents::
```
Comment (mr-c) [inline doc/dev/scripts-and-sandbox.rst]: please match the name used below

File: doc/dev/binary-file-formats.rst (hunk 3 above)
Comment (bocajnotnef) [inline doc/dev/binary-file-formats.rst]: What are the preferred extensions?

Comment (mr-c) [inline doc/dev/binary-file-formats.rst]: There's an issue on this

On Wed, Aug 12, 2015 at 3:09 PM Jake Fenton <notifications@github.com>
wrote:

> In doc/dev/binary-file-formats.rst
> <https://github.com/dib-lab/khmer/pull/1247#discussion_r36920570>:
>
> > @@ -56,6 +56,7 @@ Use Bigcount        1       6   ``0x01`` if bigcounts is used, else ``0x00``
> >  K-size              4       7   k-mer length, ``ht._ksize``. [``uint32_t``]
> >  Number of Tables    1      11   Number of Count-min Sketch tables,
> >                                  ``ht._n_tables``. [``uint8_t``]
> > +Occupied Bins       8      12   Number of occupied bins
>
> What are the preferred extensions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1247/files#r36920570>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe

**Comment by bocajnotnef** [comment]
retest this please

File: doc/dev/binary-file-formats.rst (hunk 3 above)
Comment (bocajnotnef) [inline doc/dev/binary-file-formats.rst]: Found it https://github.com/dib-lab/khmer/issues/1103

Comment (bocajnotnef) [inline doc/dev/binary-file-formats.rst]: As it stands, I just removed the recommendations, as per discussion yesterdat

**Comment by bocajnotnef** [comment]
@mr-c Updated, ready for merge

**Comment by mr-c** [comment]
LGTM, thanks!

**Pull request merged**